src/test/kotlin/org/rust/cargo/project/RsExternalLinterProjectSettingsServiceTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.project
+
+import com.intellij.testFramework.LightPlatformTestCase
+import com.intellij.util.xmlb.XmlSerializer
+import org.intellij.lang.annotations.Language
+import org.junit.internal.runners.JUnit38ClassRunner
+import org.junit.runner.RunWith
+import org.rust.cargo.project.settings.RsExternalLinterProjectSettingsService
+import org.rust.cargo.toolchain.ExternalLinter
+import org.rust.openapiext.elementFromXmlString
+import org.rust.openapiext.toXmlString
+
+@RunWith(JUnit38ClassRunner::class) // TODO: drop the annotation when issue with Gradle test scanning go away
+class RsExternalLinterProjectSettingsServiceTest : LightPlatformTestCase() {
```

Comment:
This test looks very similar to `RustProjectSettingsServiceTest` and `RustfmtProjectSettingsServiceTest`.
Maybe unify them somehow?

Reply:
I tried, I'm not sure it got better.